docs/README.md

```diff
@@ -1,9 +1,320 @@
+> **6**: Foreign goods imported directly by the seller without national equivalent as listed by Comex and natural gas.
+> **7**: Foreign goods purchased in the domestic market (within Brazil) without national equivalent as listed by Comex and natural gas.
+> **8**: National goods with imported content above 70%.
+itemDescriptor.
+
+**productType**: Fiscal type of the product according to SPED table. *Type: string*. Allowed values:
+> **FOR PRODUCT**: Finished product
+> **FOR MERCHANDISE**: Goods for resale
+> **SERVICE**: Services
+> **FEEDSTOCK**: Raw materials
+> **FIXED ASSETS**: Fixed Assets
+> **PACKAGING**: Packaging
+> **PRODUCT IN PROCESS**: Product in process
+> **SUBPRODUCT**: By-product
+> **INTERMEDIATE PRODUCT**: Intermediate product
+> **MATERIAL FOR USAGE AND CONSUMPTION**: Material for Use and Consumption
+> **OTHER INPUTS**: Other inputs
+> **NO RESTRICTION**: Other
+
+**itemDescriptor.manufacturerEquivalent**: Item is a commodity, but will be considered as a product. *Type: boolean*.
+
+**JSON Example**:
+```
+{
+    "itemCode": "12345",
+    "itemDescriptor": {
+        "taxCode": "",
+        "hsCode": "85171231"
+        "ex": "",
+        "cest": "2105301"
+        "cean": "",
+        "source": "4", "source": "4",
+        "productType": "FOR MERCHANDISE",
+        "manufacturerEquivalent": true
+    }
+}
+```
+
+This json **must** be converted to text and inserted from an integration between ERP and VTEX MasterData. It is worth noting that the skuInfo is a **string** and not an object, so all its content must be enclosed in quotes ("").
+
+Below is an example of a request (cURL) to insert a sku:
+```
+curl --location --request POST 'http://{{account}}.myvtex.com/api/dataentities/AS/documents' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "skuId": 1,
+    "skuInfo": "{\"itemCode\":\"12345\",\"itemDescriptor\":{\"taxCode\":\"\",\"hsCode\":\"85171231\",\"ex\":\"\",\"cest\":\"2105301\",\"cean\":\"\",\"source\":\"4\",\"productType\":\"FOR MERCHANDISE\",\"manufacturerEquivalent\":true}}"
+}'
+```
+
+Since these are private entities, it is also necessary to have authorization to add this information to MasterData, so authorization parameters must be placed in the header:
+
+**VtexIdclientAutCookie**: To obtain this value you must open the Developer Tools on your browser, on the Applications tab, while inside the VTEX Admin. Then look for Cookie values and copy the value of the VtexIdclientAutCookie cookie value.
+
+**AppKey/AppToken**: To get the AppKey and AppToken, log into the admin and go to the "Account Management" tab and then click on "Account". You can see all registered tokens in the security area. You can create one by clicking on "Generate appKey and appToken", then fill in the request information and click on "Generate new tokens".
+
+#### Using Avatax BR
+After finishing the setupand activating the connector your store will be able to calculate and display taxes and fees through Avalara. 
```

Comment:
I think it needs to separate words here "setup and"

Reply:
Fixed in https://github.com/vtex-apps/avatax-br-admin/pull/21/commits/c62af4ed0f97e41f7e924ef9716e410fb66f0a26